lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -89,12 +84,17 @@ public function register_routes() {
 	 * @return WP_Error|WP_REST_Response
 	 */
 	public function get_items( $request ) {
-		$args = (array) $request->get_params();
-		$args['post_type'] = $this->post_type;
-		$args['paged'] = $args['page'];
-		$args['posts_per_page'] = $args['per_page'];
+		$args                   = array();
+		$args['post_type']      = $this->post_type;
+		$args['paged']          = $request['page'];
+		$args['posts_per_page'] = $request['per_page'];
```

Comment:
I always thought `paged` and `posts_per_page` being outside of the `filter` argument was weird.  If we are going to move the WP_Query args to the filter array we should move them all!